.github/workflows/ci.yml

```diff
@@ -18,13 +18,18 @@ jobs:
       fail-fast: false
       matrix:
         version:
-          - '1.6'
           - '1' # automatically expands to the latest stable 1.x release of Julia.
         os:
           - ubuntu-latest
         arch:
           - x64
     steps:
+      - name: Set integration test flag
+        run: |
+          julia -e '
+            ENV["MLJ_TEST_INTEGRATION"]="true"'
```

Comment:
```suggestion
            open("ENV["GITHUB_ENV"]", "a") do io
                name = "MLJ_TEST_INTEGRATION"
                value = "true"
                line = "$(name)=$(value)"
                println(io, line)
            end
```

Reply:
Tried in https://github.com/alan-turing-institute/MLJ.jl/pull/1080/commits/fc139f20028e33543204c3dc5dd50451e634b3a5 but also not seeming to work.